tables/grc-international-es.utb

```diff
@@ -0,0 +1,354 @@
+#
+#  Copyright (C) 2022 Pau Moriana Bonilla, Iván Novegil Cancelas
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+#
+# -------------------------------------------------------------------------------
+#
+#  Spanish Braille for Greek Text
+#     Based on technical document B12-1 by the Comisión Braille Española (Spanish Braille Commission) https://www.once.es/servicios-sociales/braille/documentos-tecnicos/documentos-tecnicos-relacionados-con-el-braille/documentos/b12-1-texto-griego-y-su-escritura-en-braille.pdf/download
+#     Adapted and authored by Pau Moriana Bonilla <pmorianab@gmail.com> and Iván Novegil Cancelas (Spanish-speaking NVDA Community Association) <ivan.novegil@nvda.es>
```

Comment:
It would be nice to have a little more explanation/context about what the table is for (in addition to the reference to the technical document, which is great). Can you please integrate [your comment](https://github.com/liblouis/liblouis/pull/1283#issuecomment-1399996990) here?